llama_index/vector_stores/milvus.py

```diff
@@ -291,3 +313,24 @@ def query(self, query: VectorStoreQuery, **kwargs: Any) -> VectorStoreQueryResul
             ids.append(hit["id"])
 
         return VectorStoreQueryResult(nodes=nodes, similarities=similarities, ids=ids)
+
+    def _create_index_if_required(self, force: bool = False) -> None:
```

Comment:
what's the rationale behind this code @wphicks? i think it is very opinionated and prevents some use cases such as managing the index separately from llama index and just importing the index into llama index.

Reply:
> managing the index separately from llama index

Yeah, I think that is a great use case to support! We should just think about how the API should look to ensure minimal surprise when users are trying to just use this the way they would any other vector store. If you want to create a PR and tag me on it, I'd be happy to check it out.